metrics/web/lib/dashboard/presentation/widgets/strategy/metrics_value_style_strategy.dart

```diff
@@ -1,31 +1,36 @@
-// Use of this source code is governed by the Apache License, Version 2.0 
+// Use of this source code is governed by the Apache License, Version 2.0
 // that can be found in the LICENSE file.
 
+import 'package:metrics/common/presentation/metrics_theme/model/circle_percentage/attention_level/circle_percentage_attention_level.dart';
 import 'package:metrics/common/presentation/metrics_theme/model/circle_percentage/style/circle_percentage_style.dart';
 import 'package:metrics/common/presentation/metrics_theme/model/metrics_theme_data.dart';
-import 'package:metrics/common/presentation/metrics_theme/model/metrics_widget_theme_data.dart';
 import 'package:metrics/common/presentation/metrics_theme/widgets/strategy/value_based_appearance_strategy.dart';
+import 'package:metrics/dashboard/presentation/widgets/styled_circle_percentage.dart';
 
-/// A class that represents the strategy of applying the [MetricsWidgetThemeData]
-/// to the [ThemedCirclePercentage] widget based on the percent value.
+/// A class that represents the strategy of applying the [CirclePercentageStyle]
+/// to the [StyledCirclePercentage] widget based on the percent value.
 ///
-/// Gets the [MetricsWidgetThemeData] following the next rules:
+/// Gets the [CirclePercentageStyle] following the next rules:
 /// * if the percent value is 0, applies the [MetricsThemeData.inactiveWidgetTheme]
-/// * if the percent value is greater than [lowPercentBound] and less than [mediumPercentBound], applies the [MetricsCirclePercentageThemeData.lowPercentTheme]
-/// * if the percent value is greater than or equal to [mediumPercentBound] and less than [highPercentBound], applies the [MetricsCirclePercentageThemeData.mediumPercentTheme]
-/// * if the percent value is greater or equal to [highPercentBound], applies the [MetricsCirclePercentageThemeData.highPercentTheme].
+/// * if the percent value is greater than [lowPercentBound] and less than
+/// [mediumPercentBound], applies the [CirclePercentageAttentionLevel.negative]
+/// * if the percent value is greater than or equal to [mediumPercentBound] and
+/// less than [highPercentBound], applies the
+/// [CirclePercentageAttentionLevel.neutral]
+/// * if the percent value is greater or equal to [highPercentBound], applies
+/// the [CirclePercentageAttentionLevel.positive].
 class MetricsValueStyleStrategy
     implements ValueBasedAppearanceStrategy<CirclePercentageStyle, double> {
-  /// Is the lower bound of the [value]
-  /// to return the [MetricsCirclePercentageThemeData.highPercentTheme].
+  /// Is the lower bound of the given value to return the
```

Comment:
```suggestion
  /// The lower bound of the given value to return the
```
Please apply in similar docs as well.